pkgs/applications/office/espanso/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkgconfig
+, extra-cmake-modules
+, libX11
+, libXi
+, libXtst
+, libnotify
+, openssl
+, xclip
+, xdotool
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "espanso";
+  version = "0.5.5";
+
+  src = fetchFromGitHub {
+    owner = "federico-terzi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0hdc83hzcngbzdng0ia7q1hcjss42dapanaibj6i93fcir84rnqp";
+  };
+
+  cargoSha256 = "1gmysgyacnkhhpq7lybkca6gdzsivj4dgppx0k7fb17pdqwzn9nk";
+
+  nativeBuildInputs = [
+    extra-cmake-modules
+    pkgconfig
+  ];
+
+  buildInputs = [
+    libX11
+    libXtst
+    libXi
+    libnotify
+    openssl
+    xdotool
+  ];
+
+  cargoBuildFlags = [ "--locked" ];
+
+  doCheck = false;
```

```suggestion
  # Some tests require networking.
  doCheck = false;
```

It's nice to see why something is disable in the derivation, rather than having to go to the commit/PR that introduced the change.

---

Good point, I just added this.